site/content/getting-started.md

```diff
@@ -139,7 +147,7 @@ $ curl -X PUT -i -H 'Content-Type: application/json' --data-binary '{"temp": 5}'
 or (using HTTPie):
 
 ~~~sh
-$ http PUT http://localhost:8080/registration/DEFAULT_TENANT/4711 temp:=5
+$ http PUT http://localhost:8080/telemetry/DEFAULT_TENANT/4711 temp:=5
```

Comment:
I meant on the event client docs. I managed to recover it and it's now committed with c43a6c8cfa477414e4e9f06424cdbdd4ace3c71f